src/cpufreq.c

```diff
@@ -28,18 +31,52 @@
 
 static int num_cpu = 0;
 
+static char const * freq_fname = "/sys/devices/system/cpu/cpu%d/cpufreq/"
+								 "scaling_cur_freq";
+static char const * freq_fname_rhel = "/sys/devices/system/cpu/cpu%d/cpufreq/"
```

Comment:
The `_rhel` suffix sounds wrong. I don't think this is RHEL specific but rather due to a change or config option in the kernel. Can you dig up what triggered this or, else, use a more generic suffix?